src_web/kamaho-shokusu/src/Policy/TReservationInfoPolicy.php

```
@@ -35,7 +35,7 @@ public function canBulkAddSubmit(?IdentityInterface $user, TReservationInfo $res
 
     public function canBulkChangeEditSubmit(?IdentityInterface $user, TReservationInfo $resource): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$resource'. (undefined)

(UnusedFormalParameter)